app/views/govuk_publishing_components/components/_transition_countdown.html.erb

```diff
@@ -8,6 +8,7 @@
   url ||= nil
   data_attributes ||= {}
   lang ||= 'en'
+  direction ||= nil
```

Comment:
I'm in two minds about this. I can see that it's cleaner to have no direction set if there's not an explicit request for one and I can see that it's being set in another component to fix the current live bug, but it feels safer to have ltr as the default - otherwise someone might expose the same problem again by dropping this component into a page somewhere other than the related nav that turns out to run in the opposite direction.

Reply:
Good point - I've changed this so that the default is `ltr` on the countdown component rather than being set higher up. Ta!